donations/views.py

```diff
@@ -185,6 +185,10 @@ def donation_session_stripe(request):
               cancel_url=return_url_cancel,
             )
             return JsonResponse({"session_id":session.id})
+        else:
+            return JsonResponse({'errors': form.errors})
+    # If request is GET return an error 400
+    return HttpResponse(status=400)
```

Comment:
the paypal view returns a redirect to the homepage. Are we happy with the difference in behaviour?

Reply:
see comment below